presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -366,6 +367,8 @@ public void testRenameColumn()
     @Test
     public void testDropColumn()
     {
+        featuresToTest(TestedFeature.DROP_COLUMN);
```

Comment:
static imports

Reply:
another `DROP_COLUMN` is already imported